snow/consensus/snowball/binary_snowball.go

```diff
@@ -39,14 +39,11 @@ func (sb *binarySnowball) Preference() int {
 	return sb.preference
 }
 
-func (sb *binarySnowball) RecordSuccessfulPoll(choice int) {
-	sb.increasePreferenceStrength(choice)
-	sb.binarySnowflake.RecordSuccessfulPoll(choice)
-}
-
-func (sb *binarySnowball) RecordPollPreference(choice int) {
-	sb.increasePreferenceStrength(choice)
-	sb.binarySnowflake.RecordPollPreference(choice)
+func (sb *binarySnowball) RecordPoll(count, choice int) {
+	if count >= sb.alphaPreference {
+		sb.increasePreferenceStrength(choice)
```

Comment:
Should we inline this function? It used to be called twice, but now it's only called here.